### PR TITLE
HFX-1692: Backport CA-209399 and CA-210158

### PR DESCRIPTION
**Pull request by sharady**

### CODE REVIEW

**Comment by lindig** [comment]
Are these commits just cherry-picked but have been reviewed before?

File: ocaml/network/network_utils.ml [hunk 1]
```diff
@@ -751,8 +752,14 @@ module Ovs = struct
 			else
 				[]
 		in
+		let set_mac_table_size =
+			if vlan = None then
+				["--"; "set"; "bridge"; name; "other_config:mac-table-size=" ^ (string_of_int !mac_table_size)]
+			else
+				[]
+		in
 		vsctl ~log:true (del_old_arg @ ["--"; "--may-exist"; "add-br"; name] @
```
Comment (lindig) [inline ocaml/network/network_utils.ml:761]: If this is not just a backport, it would be better (more efficient, cleaner) to use `List.concat` here.

**Review by lindig — COMMENTED** [review]
Just one hint - please ignore it this is not new code but a backport.

File: ocaml/network/network_utils.ml (hunk 1 above)
Comment (sharady) [reply ocaml/network/network_utils.ml:761]: This is backport, It will be good to improve in trunk if needed.

**Review by lindig — APPROVED** [review]
This looks fine to me. Like @sharady said, my comment over `List.concat` vs. `List.append` (`@`) could be used to improve trunk.

**Pull request merged**